<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormPegawai</class>
 <widget class="QWidget" name="FormPegawai">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="formGroupBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>381</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="iDPegawaiLabel">
      <property name="text">
       <string>ID Pegawai</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="iDPegawaiLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="namaPegawaiLabel">
      <property name="text">
       <string>Nama Pegawai</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="namaPegawaiLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="alamatPegawaiLabel">
      <property name="text">
       <string>Alamat Pegawai</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="alamatPegawaiLineEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="telpPegawaiLabel">
      <property name="text">
       <string>Telp Pegawai</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="telpPegawaiLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Simpan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>230</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Ubah</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>230</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Hapus</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
